packages/Liquid-Tests.package/LQPollTests.class/instance/testStartTime.st

```diff
@@ -0,0 +1,12 @@
+tests
+testStartTime
+	| poll timeBeforeStart timeAfterStart|
+	timeBeforeStart := DateAndTime now.
+	(Delay forMilliseconds: 1) wait.
+	poll := LiquidPoll newWithPollDraft: LiquidPollDraft new.
+	poll startWithId: 'IAmAPoll' ifTaken: [self fail].
+	(Delay forMilliseconds: 1) wait.
+	timeAfterStart := DateAndTime now.
+	
+	self assert: timeBeforeStart < poll startTime equals: true.
```

Comment:
[dust] this sounds like one could add an `assert:` abstraction to better name this! `self assert: poll startTime isBetween: timeBeforeStart and: timeAfterStart`

or maybe something using intervals: `self assert: poll startTime isIn: (timeBeforeStart to: timeAfterStart)` or `self assert: ((timeBeforeStart to: timeAfterStart) includes: poll startTime)`

(code snippets are rough ideas, they most likely don't work that way)

Reply:
Good idea. I have checked what we can do, and surprisingly `DateAndTime` is a subclass of `Magnitude`, allowing us to use `between: and:` to basically write exactly what your first suggestion is.